rasa/core/processor.py

```diff
@@ -676,7 +676,7 @@ async def _run_action(
                 "Please check the logs of your action server for "
                 "more information."
             )
-            logger.debug(e, exc_info=True)
+            logger.error(str(e), exc_info=True)
```

Comment:
Why not just add `exc_info=True` to the existing error log above?

Reply:
No specific reason, was mostly adhering to the pattern [elsewhere in the Rasa codebase](https://github.com/RasaHQ/rasa/blob/master/rasa/core/channels/slack.py#L343-L345)